A3A/addons/scrt/Stringtable.xml

```diff
@@ -8748,6 +8756,30 @@
                 <Korean>%1은(는) 해당 지역에서의 우리 작전을 방해하고 있습니다. 해당 지역으로 가서 %2 이전까지 점령하십시오.</Korean>
                 <French>Un %1 perturbe nos opérations dans la région. Allez-y et capturez-le avant %2.</French>
             </Key>
+            <Key ID="STR_CON_Outpost_resources_compet_task">
+                <Original>Resource Acquisition(competition)</Original>
+                <Russian>Захват ресурсов(соревнование)</Russian>
+            </Key>
+            <Key ID="STR_CON_Outpost_resources_compet_desc">
+                <Original>A %1 would be a fine addition to our cause. Go there and capture it before %2. %3 Sent their troups as well</Original>
+                <Russian>%1 был бы прекрасным дополнением к нашему общему делу. Отправляйтесь туда и захватите его до %2. %3 также отправила туда свои войска</Russian>
+            </Key>
+            <Key ID="STR_CON_Outpost_controls_compet_task">
+                <Original>Destroy Roadblock(compet)</Original>
+                <Russian>Уничтожьте блокпост(соревнование)</Russian>
+            </Key>
+            <Key ID="STR_CON_Outpost_controls_compet_desc">
+                <Original>A %1 interferes with our operations in the area. Go there and destroy it before %2. %3 Sent their troups as well</Original>
+                <Russian>%1 мешает нашим операциям в окрестностях. Отправляйтесь туда и уничтожьте его до %2. %3 также отправила туда свои войска</Russian>
+            </Key>
+            <Key ID="STR_CON_Outpost_outposts_compet_task">
+                <Original>Take Outpost(compet)</Original>
+                <Russian>Захватите аванпост(соревнование)</Russian>
+            </Key>
```

Comment:
```suggestion
            <Key ID="STR_CON_Outpost_outposts_compet_task">
                <Original>Take Outpost (Compet)</Original>
                <Russian>Захватите аванпост (Cоревнование)</Russian>
            </Key>
```

What does "Compet" actually mean in this context btw?

Reply:
kinda "competition" between rebels and second faction(either invaders or occupants). Working title, soo... leave suggestions in the comments down below

Reply:
Frontline Struggle? instead of take outpost